docs/networking/split-horizon_config.md

```diff
@@ -0,0 +1,117 @@
+# Creating and managing Split-Horizon DNS configurations
+
+Here's how you can create, edit, and delete Split-Horizon DNS configuration for your Percona Operator for MongoDB cluster in Percona Everest.
+
+
+## Create a Split-Horizon DNS Policy
+
+Follow these steps to create a new Split-Horizon DNS policy:
+{.power-number}
+
+1.  Navigate to the Percona Everest home page and go to <i class="uil uil-cog"></i> **Settings > Policies & configurations**.
+
+    ![!image](../images/policies_page.png)
+
+2. In the **Split-Horizon DNS** section, click **Configure**. The **Split-Horizon DNS** page opens.
+
+    ![!image](../images/split-horizon_config_page.png)
+
+3. Click **Create configuration**. A pop-up window appears.
+
+    ![!image](../images/create_config_split-horizon.png)
+
+
+4. Enter the following:
+    
+    - **Name**: The desired name for your Split-Horizon policy.
+    - **Namespace**: The Kubernetes namespace where you want to create the policy.
+    - **Domain**: A domain represents how your database endpoint is identified (for example, mydb.everest.local).
```

Comment:
https://www.notion.so/percona/Split-Horizon-DNS-feature-for-PSMDB-27f674d091f3806b9d46dffaa7e24866?source=copy_link#280674d091f380afbb53cc5ec5da3923